spec/integ/crypto/crypto.spec.ts

```diff
@@ -670,18 +670,21 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
     });
 
     it("prepareToEncrypt", async () => {
-        expectAliceKeyQuery({ device_keys: { "@alice:localhost": {} }, failures: {} });
+        const homeserverUrl = "https://alice-server.com";
```

Comment:
I started to modernize a bit the tests by using E2EKeyResponder, because we are requesting keys a bit differently and the existing test were using fragile tactis `Alice will check Bob's device list (twice, because reasons)`.
I haven't done it all because it's a bit strange. In these test the olmTestDevice is sometime an alice device and sometimes a bob device.
If ok, I'd like to clean all this file in a following PR.